RELEASE.md

```diff
@@ -1,5 +1,9 @@
 # Releases #
 
+## Release 1.1.1 (2015-05-22) ##
+1. Added the new ApiCoverageHandler to log the path taken by a request to a logger named ```api-coverage-logger```.
+1. Moved the Wadl2Checker and Wadl2Dot command line utilities out of the test source tree into the main source tree. 
```

Comment:
How about saying, fixed a bug where CLI utilities were accidentally placed in test source tree :-)